frontend/src/components/print/repairPrintConfig.js

```diff
@@ -0,0 +1,27 @@
+import cloneDeep from 'lodash/cloneDeep'
+
+export default function repairConfig(
+  config,
+  camp,
+  availableLocales,
+  componentRepairers,
+  defaultContents
+) {
+  const configClone = config ? cloneDeep(config) : {}
+  if (!availableLocales.includes(configClone.language)) configClone.language = 'en'
```

Comment:
Like this it resets to english, even when the user has a language other than english.
```suggestion
  if (!availableLocales.includes(configClone.language)) configClone.language = 'en'
```

Would it be possible to pass the user locale into this function, and use this as fallback.

It is anyway not possible to change the language only for the print, so we also could not save it here, but always get it from the user.

Reply:
Yes, will do

Reply:
Fixed in [8b7538f](https://github.com/ecamp/ecamp3/pull/4764/commits/8b7538f04fe47fa55944400493c3ee6a3f7eb27e)